cmds/wifi/wifi.go

```diff
@@ -252,24 +275,44 @@ func main() {
 		return
 	}
 
+	if *show {
+		interfaces, err := scanInterfaces()
+		if err != nil {
+			log.Fatalf("error: %v", err)
+		}
+		for _, i := range interfaces {
+			fmt.Println(i)
+		}
+		return
+	}
+
 	if *test {
 		NearbyWifis = StubNearbyWifis
 		go connectWifiArbitrator()
 		startServer()
 	}
 
 	a := flag.Args()
+	if len(a) > 3 {
```

Comment:
Why?

Reply:
The Usage used to be called in main after connectWifi function has returned. However, that would print the Usage message whenever connectWifi returns an error. The message hence clutters up the screen and makes it harder to read the error while adding little use to debugging purposes. So I separate the Usage message out and only show it in the cases where the Usage message would make sense.